### PR TITLE
dx-1220 removed general transaction type from callback

### PR DESCRIPTION
**Pull request by marinafrydenb**

### CODE REVIEW

File: _includes/callback-reference.md [hunk 1]
```diff
@@ -44,6 +44,11 @@ three example scenarios of why this is important:
 *   The callback is sent from the following IP address: `82.115.146.1`
 *   A callback should return a `200 OK` response.
 
+To understand the nature of the callback, the type of transaction, its status, etc., 
+you need to perform a GET request on the received URI and inspect the response. 
+The transaction type used in the example below is `authorizations`.
+Other types of `transaction` can be `captures`, `cancellations`,`reversals`.
```
Comment (asbjornu) [inline _includes/callback-reference.md:50]: Please add a link reference to "URI Usage".

```suggestion
The transaction type or any other information can not be inferred from the URI.
See [URI usage][uri-usage] for more information.
```

File: _includes/callback-reference.md [hunk 2]
```diff
@@ -76,20 +81,14 @@ three example scenarios of why this is important:
...
-        "id": "/psp/{{ api_resource }}/payments/{{ page.payment_id }}/<transaction type>/{{ page.transaction_id }}",
+        "id": "/psp/{{ api_resource }}/payments/{{ page.payment_id }}/authorizations/{{ page.transaction_id }}",
         "number": 333333333
     }
 }
 ```
 
 {% endif %}
 
-{:.table .table-striped}
```
Comment (asbjornu) [inline _includes/callback-reference.md:87]: Let's add examples of a response for an authorization request here. I think we have an include for that.

**Review by asbjornu — CHANGES_REQUESTED** [review]

File: _includes/callback-reference.md (hunk 2 above)
Comment (marinafrydenb) [reply _includes/callback-reference.md:87]: added!

**Review by arebra — APPROVED** [review]
Top notch!

File: _includes/callback-reference.md [hunk 3]
```diff
@@ -76,19 +81,17 @@ three example scenarios of why this is important:
...
-| Parameter            | Description                                                |
-| :------------------- | :--------------------------------------------------------- |
-| `<transaction type>` | `authorizations`, `captures`, `cancellations`,`reversals` |
-| `<payment instrument>` | `creditcard`, `invoice`, `swish`, `vipps`,`mobilepay`, `callback` |
+{% include transaction-response.md api_resource=include.api_resource
```
Comment (asbjornu) [inline _includes/callback-reference.md:92]: Please add some text to introduce the transaction response. Something like:

> When doing an HTTP request towards the `transaction.id` found in the callback, you will receive a transaction response as described below.

**Review by asbjornu — CHANGES_REQUESTED** [review]

File: _includes/callback-reference.md [hunk 4]
```diff
@@ -44,6 +44,11 @@ three example scenarios of why this is important:
 *   The callback is sent from the following IP address: `82.115.146.1`
 *   A callback should return a `200 OK` response.
 
+To understand the nature of the callback, the type of transaction, its status, etc.,
+you need to perform a GET request on the received URI and inspect the response.
+The transaction type or any other information can not be inferred from the URI.
+See [URI usage][uri-usage] for more information.
```
Comment (asbjornu) [inline _includes/callback-reference.md:50]:
```suggestion
you need to perform a `GET` request on the received URI and inspect the response.
The transaction type or any other information can not and should not be inferred
from the URI. See [URI usage][uri-usage] for more information.
```

File: _includes/callback-reference.md [hunk 5]
```diff
@@ -76,19 +81,20 @@ three example scenarios of why this is important:
...
 
 {% endif %}
 
-{:.table .table-striped}
-| Parameter            | Description                                                |
-| :------------------- | :--------------------------------------------------------- |
-| `<transaction type>` | `authorizations`, `captures`, `cancellations`,`reversals` |
-| `<payment instrument>` | `creditcard`, `invoice`, `swish`, `vipps`,`mobilepay`, `callback` |
+To acquire the transaction response example shown below, you can do a
+HTTP request towards the `transaction.id` found in the callback resource.
```
Comment (asbjornu) [inline _includes/callback-reference.md:93]:
```suggestion
When performing an HTTP `GET` request towards the URI found in the
`transaction.id` field of the callback, the response is going to look 
something like the abbreviated example provided below.
```

**Review by asbjornu — CHANGES_REQUESTED** [review]

**Review by asbjornu — APPROVED** [review]

**Pull request merged**